src/DockLayout.tsx

```diff
@@ -137,8 +137,9 @@ class DockPortalManager extends React.PureComponent<LayoutProps, LayoutState> {
     let cache = this._caches.get(id);
     if (!cache) {
       let div = document.createElement('div');
+      let pendingDestroy = false;
       div.className = 'dock-pane-cache';
-      cache = {div, id, owner};
+      cache = {div, id, owner, pendingDestroy};
       this._caches.set(id, cache);
     } else {
       cache.owner = owner;
```

Comment:
should we also set the `pendingDestroy` false here? 

I think the owner changes when the tabs are dragged-dropped into different panels. When dragged the `removeCache` is called and on drop `getCache` is called.

Reply:
Should be resolved